src/AutoMerge/guidelines.jl

```diff
@@ -322,6 +322,63 @@ function meets_name_match_check(
     return (true, "")
 end
 
+# This check checks for an explanation of why a breaking change is breaking
+const guideline_breaking_explanation = Guideline(;
+    info = "Release notes have not been provided that explain why this is a breaking change.",
+    docs = "If this is a breaking change, release notes must be given that explain why this is a breaking change (i.e. mention \"breaking\"). To update the release notes, please see the \"Providing and updating release notes\" subsection under \"Additional information\" below.",
+    check=data -> meets_breaking_explanation_check(data))
+
+function meets_breaking_explanation_check(data::GitHubAutoMergeData)
+    # Look up PR here in case the labels are slow to be applied by the Registrator bot
+    # which decides whether to add the BREAKING label
+    pr = GitHub.pull_request(data.api, data.registry, data.pr.number; auth=data.auth)
+    return meets_breaking_explanation_check(pr.labels, pr.body)
+end
+function meets_breaking_explanation_check(labels::Vector, body::AbstractString)
+    if any(==("BREAKING"), labels)
+        release_notes = get_release_notes(body)
+        example_detail = """
```

Comment:
I'm a little worried about the indentation here, I remember that being a pain w/ the compat one, where detail block wouldn't render within the list itself

Reply:
that one is https://github.com/JuliaRegistries/RegistryCI.jl/blob/master/test/reference_comments/comment_pass_false_type_new_package_suggest_onepointzero_false_version_1.0.0_point_to_slack_false.md#2-automerge-guidelines-which-are-not-met-

populated in the tests via https://github.com/JuliaRegistries/RegistryCI.jl/blob/11d89a142313fb8516b8351c57b3035956a8240a/test/automerge-unit.jl#L72

Reply:
I indented them. If that's not what you're saying, and it's clear what to do please feel free to push

Reply:
sure, I pushed some commits, now it's in the reference tests so I can check how it formats: https://github.com/JuliaRegistries/RegistryCI.jl/blob/34367b742375cb4670bb310b895b0cf33c22fc52/test/reference_comments/comment_pass_false_type_new_package_suggest_onepointzero_true_version_0.1.0_point_to_slack_false.md

<img width="1021" alt="Screenshot 2024-12-16 at 22 28 27" src="https://github.com/user-attachments/assets/2b5abbcf-ef67-48d0-ad20-0d838eb47261" />